streaming/src/main/scala/org/apache/spark/streaming/ui/BatchPage.scala

```diff
@@ -37,10 +37,14 @@ private[ui] class BatchPage(parent: StreamingTab) extends WebUIPage("batch") {
   private def columns: Seq[Node] = {
     <th>Output Op Id</th>
       <th>Description</th>
-      <th>Output Op Duration</th>
+      <th>Output Op Duration {SparkUIUtils.tooltip("Time taken for all the jobs of this batch to" +
```

Comment:
I'm not sure users would know what those delays are. It's hard to describe it succinctly. I think the key point is that it is "Time from job submission until all are completed" This clarifies it isn't just since the time they started processing.

But, that kind of sounds like what Job Duration is. How is Output Op duration different? the description sounds the same.

Reply:
@iRakson I don't think this is worth a lot of back and forth. If you can update, great, otherwise I'm going to close it.

Reply:
Output Op duration is the time taken for all the jobs of that batch to finish processing from the time they were submitted. So it gives scheduling delay(time to schedule first job of the batch) + processing delay(time from start of processing of first job till the end of processing of last job of batch).

While job duration gives detail about a single job, Output Op duration gives info about the entire batch.